metacatalog/models/variable.py

```diff
@@ -151,14 +151,12 @@ def to_dict(self, deep=False) -> dict:
             name=self.name,
             symbol=self.symbol,
             unit=self.unit.to_dict(deep=False),
-            column_names=self.column_names,
-            keyword=self.keyword.to_dict(deep=False)
+            column_names=self.column_names
         )
 
         # set optionals
-        for attr in ('keyword'):
-            if hasattr(self, attr) and getattr(self, attr) is not None:
-                d[attr] = getattr(self, attr).to_dict(deep=False)
+        if self.keyword:
+            d['keyword'] = self.keyword.to_dict(deep=False) 
```

Comment:
Hmm, to me it seems like the keyword should have been part of the returned dictionary in the first place. Did that not work beforehand?
I can't see why the new code should work and the old not. Additionally, I would prefer the old code. We should investigate this, before we merge.

Reply:
I found the bug in the old code!  
```python
for attr in ('keyword'):
    print(attr)
```
actually loops over the letters in `keyword`, output from above:
```python
k
e
y
w
o
r
d
```

Replacing the `tuple` (which is actually of type `str` when checking) with a `list` does solve the error and makes the old code work (which I also like more):

```python
for attr in ['keyword']:
    print(attr)
```
```python
keyword
```

Reply:
Perfect! Thanks!